Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -517,11 +517,42 @@ public override void Update()
             if (!IsSetupValid) { return; }
 
             UpdatePointers();
+
+            if (gazeProviderPointingData?.Pointer != null)
+            {
+                UpdateGazeProvider();
+            }
             UpdateFocusedObjects();
 
             PrimaryPointer = primaryPointerSelector?.Update();
         }
 
+        /// <summary>
+        /// Update the hit information for the gaze provider.
+        /// 
+        /// For the sake of StabilizationPlaneModifier and potentially other 
+        /// components that care where the user's looking, we need to do a gaze 
```

Comment:
Nit: Generally for comments, avoid mentioning what or who is calling you, as it's possible for this comment to get out of date just via that component changing (and it's super easy to not update this comment, as you could make it to StabilizationPlaneModifierdoesn't call this anymore, but unless you grep StabilizationPlaneModifier (i.e. not just the functions you changed in StabilizationPlaneModifier), you won't think to update this)

So long story short, probably just the "Updates the the gaze raycast provider even in scenarios where gaze isn't used for focus"